tests/test_openai.py

```diff
@@ -0,0 +1,99 @@
+from typing import List
+import pytest
+import jinja2
+import httpx
+from pytest_httpx import HTTPXMock
+
+from recipes.openai import OpenAISuggester, get_api_credentials
+
+
+# Setup the template and the suggester
+environment = jinja2.Environment()
+template = environment.from_string("Prompt: {{ text }}")
+
+openai = OpenAISuggester(
+    prompt_template=template,
+    labels=["label1", "label2"],
+    max_examples=1,
+    segment=False,
+    openai_model="text-davinci-003",
+    openai_api_key="Fake api key",
+    openai_api_org="Fake api org",
+    response_parser=lambda x: {"key": "value"},
+    openai_n_retries=1,
+    openai_read_timeout_s=1,
+    openai_retry_timeout_s=1,
+)
```

Comment:
Might be good to define `openai` as a fixture here?

Reply:
I thought of creating a fixture here, but it's unnecessary complexity: creating a module scope that works within this file is already accomplished by defining the class.